src/how-to/render/display-text.md

```diff
@@ -1,5 +1,164 @@
 # Displaying Text
 
-(Stub) This page covers how text displaying works. It will cover `Line`, `Span`, `Text`, the
-migration of `Spans` to `Text`, and how these can be created, styled, displayed, altered, and such.
-These are all interrelated topics so, they are on the same page.
+This page covers how text displaying works. It will cover `Span`, `Line`, and `Text`, and how these
+can be created, styled, displayed, altered, and such.
+
+## `Span`
+
+A `Span` is a styled segment of text. You can think of it as a substring with its own unique style.
+It is the most basic unit of displaying text in `ratatui`.
+
+The examples below assume the following imports:
+
+```rust
+use ratatui::{prelude::*, widgets::*};
+pub type Frame<'a> = ratatui::Frame<'a, CrosstermBackend<std::io::Stderr>>;
+```
+
+A `Span` consists of "content" and a "style" for the content. And a `Span` can be created in a few
+different ways.
+
+1. using `Span::raw`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::raw("This is text that is not styled");
+    // -- snip ---
+}
+```
+
+2. using `Span::styled`:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = Span::styled("This is text that will be yellow", Style::default().fg(Color::Yellow));
+    // -- snip ---
+}
+```
+
+3. using the `Stylize` trait:
+
+```rust
+fn ui(_app: &App, f: &mut Frame<'_>) {
+    let span = "This is text that will be yellow".yellow();
+    // -- snip ---
+}
+```
+
+`Span`s are the building blocks to build styled content that is text that is represented
+horizontally.
+
+## `Line`
+
+The next building block that we are going to talk about is a `Line`. A `Line` represents a single
+"line" of text. The `Line` struct is essentially a `Vec<Span>`.
```

Comment:
maybe make it clear here that it ends with a newline? So it could read:

"The `Line` struct is essentially a `Vec<Span>` with a newline (`\n`) character at the end."

I think the stress should be that a `Line` is different from a `Span` because it represents a full line, but different from a `Text` because it is *only* one line.

Reply:
I don't think it necessarily ends with a new line, right? There's nothing about a new line in the `Line` struct definition. It is just a collection of spans. A line can be displayed wrapped or not, and the newline (`\n`) would be placed based on the available width. 

This is what we want to convey:

`Span` - text that contains just one style
`Line` - Collection of `Span`s
`Text` - Collection of `Line`s

Reply:
That is a fair point. I think the part that made me stop while reading was the " represents a "line" of text" and my first thought was why is it a "line" and not just a ... line of text.

Reply:
I updated that section to add more clarity. Let me know what you think.